echopype/tests/utils/test_processinglevels_integration.py

```diff
@@ -129,5 +129,5 @@ def _freqdiff_applymask(test_ds):
     # No product level is assigned because at present compute_MVBS drops the lat/lon data
     # associated with the input Sv dataset
     # ds = ds.rename_vars(name_dict={"Sv": "Sv_unmasked", "Sv_ch0": "Sv"})
-    mvbs_ds = ep.commongrid.compute_MVBS(ds, range_meter_bin=30, ping_time_bin='1min')
+    mvbs_ds = ep.commongrid.compute_MVBS(ds, range_meter_bin="30m", ping_time_bin='1min')
     _absence_test(mvbs_ds)
```

Comment:
Now that `compute_MVBS` can preserve lat-lon (YAY!!), a processing level attribute is expected.
```suggestion
    _presence_test(mvbs_ds, "Level 3B")
```

Reply:
I'll take the liberty of committing my "suggestion" ...